src/main/resources/db/changelog/changes/08-insert-categories.yaml

```
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert_categories
+      author: alex
```

```suggestion
      author: alex_liubchenko
```
it's better to pass also last name